src/ServiceControl.Persistence.Tests/Recoverability/EditMessageTests.cs

```diff
@@ -20,10 +20,15 @@ sealed class EditMessageTests : PersistenceTestBase
     {
         EditHandler handler;
         readonly TestableUnicastDispatcher dispatcher = new();
+        readonly ErrorQueueNameCache errorQueueNameCache = new()
+        {
+            ResolvedErrorAddress = "TestAddress"
```

Comment:
```suggestion
            ResolvedErrorAddress = "errorQueueName"
```

This is a total nitpick but IMHO it makes the intent of the value clearer to people reading the tests.